<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_gestaovendas</class>
 <widget class="QDialog" name="frm_gestaovendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestão de Vendas</string>
  </property>
  <widget class="QPushButton" name="btn_mostratTodasVendas">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>35</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Todas as Vendas</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listaProdutos">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>363</y>
     <width>711</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_relatorio">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>641</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar Relatório </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Data Inicial:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="de_datainicial">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>10</y>
     <width>109</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Data Final:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="de_datafinal">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>67</y>
     <width>736</width>
     <height>260</height>
    </rect>
   </property>
   <property name="title">
    <string>Vendas:</string>
   </property>
   <widget class="QTableWidget" name="tw_listaVendas">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>31</y>
      <width>711</width>
      <height>220</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>333</y>
     <width>736</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Produtos:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Filtrar:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_filtrar"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>366</x>
     <y>34</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_filtrar">
      <property name="text">
       <string>Filtrar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>btn_mostratTodasVendas</zorder>
  <zorder>tw_listaProdutos</zorder>
  <zorder>btn_relatorio</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
